PythonTool/OperatorSlice.py

```diff
@@ -0,0 +1,50 @@
+from visit import *
+
+
+def OperatorSlice(File, OperSet, myList, Centroids, SinglePlane=1):
+    """
+    with no args - default direction and location
+    [(<x,y,z>,val),(<x,y,z>,val),....]
+    produces one image for each tuple with slice at plane <x,y,z>=val
+    """
+
+    ResetView()
```

Comment:
Some comments on what these calls are  and the parameters being passed to them would be helpful :)